pkg/fdbadminclient/admin_client.go

```diff
@@ -45,6 +45,10 @@ type AdminClient interface {
 	// database.
 	GetExclusions() ([]fdbv1beta2.ProcessAddress, error)
 
+	// GetExclusionsFromStatus gets a list of the addresses currently excluded from the
+	// database using status.
+	GetExclusionsFromStatus(status *fdbv1beta2.FoundationDBStatus) ([]fdbv1beta2.ProcessAddress, error)
```

Comment:
Is there a reason to add a new method here and not just use his method: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/pkg/fdbstatus/status_checks.go#L251?

Reply:
I needed this method to perform mocking. The real implementation of this function is calling the method you mentioned while the mocked version is used in unit tests that returns the ExcludedAdresses. Directly calling the `fdbstatus.GetExclusion()` method will break many unit tests as and various places of the mocked code we are adding excluded servers / ips in the `ExcludedAddresses` field of mocked admin client.

Reply:
I would prefer not to add another public exposed method to the admin client only to get some unit tests passed. From my perspective the correct approach would be to adjust the `GetStatus` method in the mock client to include the excluded servers in the generated status. That should solve the issue with the failing unit tests and will not require to expose another method with the benefit that the mocked output is more realistic.